GoogleDataTransport/GDTCORTests/Unit/Helpers/GDTCOREventGenerator.m

```diff
@@ -15,10 +15,11 @@
  */
 
 #import "GDTCORTests/Unit/Helpers/GDTCOREventGenerator.h"
+#import "GDTCORTests/Unit/Helpers/GDTCORDataObjectTesterClasses.h"
```

Comment:
Move this down just below or just above the `#import "GDTCORLibrary/Private/GDTCOREvent_Private.h"`. This first line (or sometimes 2 lines) of import should be reserved for the corresponding header and/or private header.

http://go/objc-style#order-of-includes